occrp/models.py

```diff
@@ -65,14 +65,14 @@ class Source(db.Model):
 class Event(db.Model):
     __tablename__ = 'event'
     id = Column(Integer, primary_key=True)
-    title = Column(String, nullable=False, unique=True)
+    description = Column(String, nullable=False, unique=True)
```

Comment:
It doesn't look like the unique constraint for the description column is part of the migration. Is that on purpose? Do you care if that's actually checked at the database level?

Reply:
Is it not here? https://github.com/datamade/occrp-timeline-tool/pull/26/files#diff-d6dc455171d519ce278427ff03eeff40 (line 24) Or am I misreading this?